### PR TITLE
feat(kicon): add connection icon to kicons

### PR DESCRIPTION
**Pull request by xitongzou**
### Summary
Adding the connection icon to Kicons

#### Changes made:
*Adding the connection icon to Kicons

### PR Checklist
- [x] Does not introduce dependencies
- [x] **Functional:** all changes do not break existing APIs and if so, bump major version.
- [x] **Tests pass:** check the output of yarn test packages/<Kongponent>
- [x] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
- [x] **Framework style:** abides by the essential rules in Vue's style guide
- [x] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs), or leftover comments
- [x] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

**Comment by xitongzou** [comment]
also, checked that it was under 1kb

File: packages/KIcon/icons/icn-connections.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="18px" height="24px" viewBox="0 0 18 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>Combined Shape</title>
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" opacity="0.7">
+        <path d="M14,0 L18,4 L18,22.0092049 C18,23.1086907 17.1073772,24 16.0049107,24 L1.99508929,24 C0.893231902,24 6.75015599e-14,23.1029738 6.75015599e-14,21.9950534 L6.75015599e-14,2.00494659 C6.75015599e-14,0.897645164 0.897060126,0 2.00585866,0 L14,0 Z M13,2 L2,2 L2,22 L16,22 L16,5 L13,5 L13,2 Z M14,16 L14,17.5 L4,17.5 L4,16 L14,16 Z M14,12 L14,13.5 L4,13.5 L4,12 L14,12 Z M14,8 L14,9.5 L4,9.5 L4,8 L14,8 Z" id="Combined-Shape" fill="#ACCEE6"></path>
+    </g>
+</svg>
```
Comment (jeffyongtaotang) [inline packages/KIcon/icons/icn-connections.svg:7]:
```suggestion
</svg>

```

File: packages/KIcon/icons/icn-connections.svg [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="18px" height="24px" viewBox="0 0 18 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>Combined Shape</title>
```
Comment (darrenjennings) [inline packages/KIcon/icons/icn-connections.svg:3]: omit the title so user's don't see this text when hovering over the icon. KIcon uses the icon name e.g. 'connections' when rendering.

**Comment by darrenjennings** [comment]
Curious what the background on this icon is... What is "connections"? It's an icon of a piece of paper? Where will we use it?

File: packages/KIcon/icons/icn-connections.svg [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (travispamaral) [inline packages/KIcon/icons/icn-connections.svg:1]: Lets ditch the XML type, we don't use this in any kongponents

**Comment by xitongzou** [comment]
not using any connection icons for now so closing

**Pull request closed without merging**